README.md

```diff
@@ -51,18 +53,28 @@ conda install -c conda-forge jupyterlab
 
 #### pip
 
-If you use [`pip`](https://pypi.org/project/jupyterlab/), you can install it as:
+pip is a package management system for installing and updating Python packages. pip comes with Python, so you get pip simply by installing Python. On Ubuntu and Fedora Linux, you can simply use your system package manager to install the `python3-pip` package. [_The Hitchhiker's Guide to Python_ provides some guidance on how to install Python on your system if it isn't already](https://docs.python-guide.org/starting/installation/); you can also [install Python directly from python.org](https://www.python.org/getit/). You might want to [upgrade pip](https://pip.pypa.io/en/stable/installing/) before using it to install other programs.
 
-```bash
-pip install jupyterlab
-```
+JupyterLab is implemented in Python, and as of July 2018 JupyterLab can run on either Python2 or Python3. However, it's recommended that you install JupyterLab on Python3, even when running Python2 code - and there are different ways to do that. Therefore:
```

Comment:
"JupyterLab uses Python" (more correct, since the vast majority of jlab code is typescript...)